js/app/courseHistoryStore.js

```diff
@@ -85,7 +85,7 @@ const CourseHistoryStore = new GObject.Class({
         Eknc.Engine.get_default().query_promise(query)
         .then((results) => {
             if (results.models.length > 0) {
-                this.set_current_subset(results[0]);
+                this.set_current_subset(results.models[0]);
```

Comment:
@ptomato https://github.com/endlessm/eos-knowledge-lib/commit/944f983b93814a99f9b164651c06767deb7004d9

Looks like at some point we moved from `results` being the array of models itself, to it being an object with the `models` property on it. I guess we didn't catch it because only capacitate app hits this code path and we didn't end up releasing capacitate.

In any case, your fix is correct :-)